src/background/modules/AutocorrectHandler.js

```diff
@@ -26,6 +26,76 @@ let antipatterns = [];
 // Adapted from: https://github.com/mozilla/webextension-polyfill/blob/master/src/browser-polyfill.js
 const IS_CHROME = Object.getPrototypeOf(browser) !== Object.prototype;
 
+/**
+ * Traverse Trie tree of objects to create RegEx.
+ *
+ * @param {Object.<string, Object|boolean>} obj
+ * @returns {string}
+ */
+function pattern(obj) {
```

Comment:
Instead of `obj` (that could be anything) maybe better call it `trieTree` or so, if that is the expected input, actually, (not _any_ kind of object).

Reply:
Good idea, will do...